src/helpers/files.js

```diff
@@ -224,28 +225,30 @@ function parseGitIgnore (projectRoot) {
  * @param {string[]} arrayOfFiles
  * @returns {string[]}
  */
-function getAllFiles (projectRoot, dirPath, arrayOfFiles = []) {
+function getAllFiles (projectRoot, dirPath, args, arrayOfFiles = []) {
+  log(`Searching for files in ${dirPath}`, { level: 'debug', args })
   const files = fs.readdirSync(dirPath)
 
   files.forEach(function (file) {
+    if (isBlacklisted(projectRoot, file, manualBlacklist())) {
+      return
+    }
     if (
-      fs.statSync(path.join(dirPath, file)).isDirectory() &&
-      !isBlacklisted(projectRoot, file, manualBlacklist())
+      fs.statSync(path.join(dirPath, file)).isDirectory()
     ) {
       arrayOfFiles = getAllFiles(
         projectRoot,
         path.join(dirPath, file),
+        args,
         arrayOfFiles
       )
     } else {
-      if (!isBlacklisted(projectRoot, file, manualBlacklist())) {
-        arrayOfFiles.push(
+      arrayOfFiles.push(
           `${path.join(dirPath.replace(projectRoot, '.'), file)}\n`
-        )
-      }
+      )
     }
   })
-
+  log(`Search complete for files in ${dirPath}`, { level: 'debug', args })
```

Comment:
nit: not sure we need to specify again if we have the `dirPath` at the beginning of this method. doesn't matter ot me either way

Reply:
Because it's a nested recursive method.